src/debug.c

```diff
@@ -498,6 +499,8 @@ void debugCommand(client *c) {
 "    Output SHA and content of all scripts or of a specific script with its SHA.",
 "MARK-INTERNAL-CLIENT [UNMARK]",
 "    Promote the current connection to an internal connection.",
+"ASM-FAILPOINT <channel> <state>",
+"    Set a fail point for the specified channel and state.",
```

Comment:
perhaps, we can add a few words like "(for cluster atomic slot migration)".  So, others can figure out the context without checking the code.